lesson_17_psycopg2/data_insert_table.py

```diff
@@ -0,0 +1,34 @@
+from config import user, db_name, password, host
+import psycopg2
+
+
+def user_input():
+    try:
+        connection = None
+        connection = psycopg2.connect(
+            host=host,
+            database=db_name,
+            user=user,
+            password=password
+        )
+
+        with connection.cursor() as cursor:
+            cursor.execute(
+                """select food_name from food_info"""
```

Comment:
here I can see only select .. it is getting data
But you called file like `insert`